src/gui/guiFormSpecMenu.cpp

```diff
@@ -781,6 +782,54 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 	errorstream<< "Invalid image element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if ((parts.size() == 3) ||
+		((parts.size() > 3) && (m_formspec_version > FORMSPEC_API_VERSION)))
+	{
```

Comment:
```cpp
	if (parts.size() != 3 &&
			(parts.size() < 3 || m_formspec_version <= FORMSPEC_API_VERSION)) {
		errorstream << "Invalid animated image element(" << parts.size() << "): '" << element << "'"  << std::endl;
		return;
	}
...
```

Reply:
I like that, but it doesn't follow the existing convention used in every other `parseX()` function, as far as I can tell. I'll make the change if another reviewer agrees with this.

Reply:
Actually, I'm just going to be the change I want to see in the world and do it. If anyone disagrees in later reviews, I can always revert the commit.